client/src/index.jsx

```diff
@@ -1,16 +1,73 @@
 const React = require('react');
 const ReactDOM = require('react-dom');
+const axios = require('axios');
+import ProjectOwner from './components/ProjectOwner.jsx';
+import PledgeOption from './components/PledgeOption.jsx';
 
 class Pledge extends React.Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
+    this.state = {
+      owner: {
+        name: 'Name',
+        created: 0,
+        backed: 0,
+        aboutMe: 'About Me'
+      },
+      options: []
+    }
+    this.fetchOwner = this.fetchOwner.bind(this);
+    this.fetchOptions = this.fetchOptions.bind(this);
+  }
+
+  componentDidMount() {
+    let itemId = Number(window.location.pathname.replaceAll('/', ''));
```

Comment:
could this variable name be more specific? It looks like this will be the endpoint as well as the itemId